### PR TITLE
2. Improves role pages boilerplate

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

File: src/content/pages/earn/token-holder.md [hunk 1]
```diff
@@ -2,4 +2,34 @@
...
+secondaryInfo:
+  left:
+    label: T Token Holders
+    title: Participate in Threshold Governance!
+    description: As a T token holder, you can participate in the Threshold DAO and
+      vote to decide what's next for the network.
+    buttons:
+      - label: About Governance
+        url: /governance
+        variant: EXTERNAL_OUTLINE
```
Comment (r-czajkowski) [inline src/content/pages/earn/token-holder.md:25]: Should't be `INTERNAL_OUTLINE` because it is an internal link?

File: static/admin/config.yml [hunk 2]
```diff
@@ -743,7 +743,77 @@ collections:
...
+                  {
+                    label: "Title",
+                    name: "title",
+                    widget: "string",
+                    minimal: true,
+                  },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "markdown",
```
Comment (r-czajkowski) [inline static/admin/config.yml:769]: Please see https://github.com/threshold-network/splash-page/pull/23/#discussion_r844042937

File: static/admin/config.yml [hunk 3]
```diff
@@ -761,7 +831,40 @@ collections:
...
```
Comment (r-czajkowski) [inline static/admin/config.yml:857]: Same as above

File: static/admin/config.yml [hunk 4]
```diff
@@ -1122,7 +1225,216 @@ collections:
...
```
Comment (r-czajkowski) [inline static/admin/config.yml:1251]: Same as above.

File: static/admin/config.yml [hunk 5]
```diff
@@ -1122,7 +1225,216 @@ collections:
...
+                        {
+                          label: "Title",
+                          name: "title",
+                          widget: "string",
+                          minimal: true,
+                        },
+                        {
+                          label: "Description",
+                          name: "description",
+                          widget: "markdown",
```
Comment (r-czajkowski) [inline static/admin/config.yml:1330]: Same as above.

File: src/templates/earn-page/RolesNav.tsx [hunk 6]
```diff
@@ -2,10 +2,12 @@ import { FC } from "react"
 import { Button, HStack, StackProps } from "@chakra-ui/react"
 import { Link as GatsbyLink } from "gatsby"
 import { Location } from "@reach/router"
+import useChakraBreakpoint from "../../hooks/useChakraBreakpoint"
 
 // @ts-ignore
 const RolesNav = ({ location, ...props }) => {
   const { pathname } = location
+  const smallScreen = useChakraBreakpoint("md")
```
Comment (r-czajkowski) [inline src/templates/earn-page/RolesNav.tsx:10]: Can we rename `isSmallScreen`?

File: static/admin/config.yml [hunk 7]
```diff
@@ -1122,7 +1225,216 @@ collections:
...
```
Comment (r-czajkowski) [inline static/admin/config.yml:1393]: Same as above

File: src/content/pages/earn/token-holder.md (hunk 1 above)
Comment (georgeweiler) [reply src/content/pages/earn/token-holder.md:25]: great catch

File: src/templates/earn-page/RolesNav.tsx (hunk 6 above)
Comment (georgeweiler) [reply src/templates/earn-page/RolesNav.tsx:10]: of course. updated.

**Pull request merged**